pkgs/development/tools/literate-programming/noweb/default.nix

```diff
@@ -1,29 +1,74 @@
-{stdenv, fetchurl, gawk}:
-
-stdenv.mkDerivation {
-  name = "noweb-2.11b";
-  src = fetchurl {
-    urls = [ "http://ftp.de.debian.org/debian/pool/main/n/noweb/noweb_2.11b.orig.tar.gz"
-             "ftp://www.eecs.harvard.edu/pub/nr/noweb.tgz"
-          ];
-    sha256 = "10hdd6mrk26kyh4bnng4ah5h1pnanhsrhqa7qwqy6dyv3rng44y9";
+{ stdenv, fetchFromGitHub, gawk, groff, icon-lang ? null }:
+
+let noweb = stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
This line is unnecessary, `stdenv.mkDerivation` already defines `name = "${pname}-${version"};` if both `pname` and `version` are defined.

Reply:
Good catch, thanks. I originally wrote this a before that was the case.